apps/frontend/src/lib/api/courses.ts

```diff
@@ -30,6 +30,20 @@ export interface ICourse {
   title: string;
   toDate: string;
   typicallyOffered: Semester[] | null;
+  aggregatedRatings?: {
+    __typename?: "AggregatedRatings";
```

Comment:
I would recommend typing `aggregatedRatings` and `metrics` within the `ratings.ts` file and importing it here since it is used in multiple places